### PR TITLE
Added converter script and sudo in the right places and sort afew typos

### PR DESCRIPTION
**Pull request by dinger1986**

### CODE REVIEW

**Comment by dinger1986** [comment]
@rustdesk changed the paths to /var/log/rustdesk-server

File: install.sh [hunk 1]
```diff
@@ -164,8 +164,8 @@ WorkingDirectory=/var/lib/rustdesk-server/
 User=${uname}
 Group=${uname}
 Restart=always
-StandardOutput=append:/var/log/rustdesk/rustdeskhbbs.log
-StandardError=append:/var/log/rustdesk/rustdeskhbbs.error
+StandardOutput=append:/var/log/rustdesk-server/rustdeskhbbs.log
+StandardError=append:/var/log/rustdesk-server/rustdeskhbbs.error
```
Comment (rustdesk) [inline install.sh:168]: Sorry, I did not review contributors' code seriously before. I just made a change. let's keep consistent. :(
https://github.com/rustdesk/rustdesk-server/commit/b713303c1557f158a3b6279fcce5139190ed2937

File: update.sh [hunk 2]
```diff
@@ -12,8 +12,8 @@ if [ $RDLATEST == $RDCURRENT ]; then
     exit 0
 fi
 
-sudo systemctl stop rustdesk-hbbs.service
-sudo systemctl stop rustdesk-hbbr.service
+sudo systemctl stop rustdeskhbbs.service
+sudo systemctl stop rustdeskhbbr.service
```
Comment (rustdesk) [inline update.sh:16]: Not consistent with https://github.com/rustdesk/rustdesk-server/blob/b713303c1557f158a3b6279fcce5139190ed2937/debian/rustdesk-server-hbbr.install#L2

File: convertfromos.sh [hunk 3]
```diff
@@ -0,0 +1,296 @@
+#!/bin/bash
+
+# This script will do the following to install RustDesk Pro replacing RustDesk Opensource
+# 1. Disable and removes the old services
+# 2. Install some dependencys
```
[the remark this answers is not in the document]
Comment (dinger1986) [reply convertfromos.sh:5]: Dependencys?

File: update.sh [hunk 4]
```diff
@@ -82,36 +82,36 @@ if [ "${ARCH}" = "x86_64" ] ; then
...
+sudo chmod +x /usr/bin/hbbr
 
-sudo systemctl start rustdesk-hbbs.service
-sudo systemctl start rustdesk-hbbr.service
+sudo systemctl start rustdeskhbbs.service
+sudo systemctl start rustdeskhbbr.service
 
 while ! [[ $CHECK_RUSTDESK_READY ]]; do
-  CHECK_RUSTDESK_READY=$(sudo systemctl status rustdesk-hbbr.service | grep "Active: active (running)")
+  CHECK_RUSTDESK_READY=$(sudo systemctl status rustdeskhbbr.service | grep "Active: active (running)")
```
[the remark this answers is not in the document]
Comment (dinger1986) [reply update.sh:114]: I'll test again but the script seemed to be ignoring them as it had the - in it so removed them.

I'll test again and sort it

File: convertfromos.sh [hunk 5]
```diff
@@ -0,0 +1,296 @@
...
+# 3. Setup ufw firewall if available
+# 4. Create a Folder /var/lib/rustdesk-server and copy the certs here
```
[the remark this answers is not in the document]
Comment (dinger1986) [reply convertfromos.sh:7]: Yes of course it's not copying across

File: install.sh (hunk 1 above)
Comment (dinger1986) [reply install.sh:168]: Sure

File: update.sh (hunk 2 above)
Comment (dinger1986) [reply update.sh:16]: That's the - I'll check and sort this

File: install.sh (hunk 1 above)
Comment (dinger1986) [reply install.sh]: so thats in both install.sh and convertfromos.sh both are resolved now (hbbr and hbbs log and error)

File: convertfromos.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (dinger1986) [reply convertfromos.sh]: have replaced the word dependencys with dependencies and also changed to RustDesk Server Pro in install.sh and convertfromos.sh

File: update.sh (hunk 4 above)
[the remark this answers is not in the document]
Comment (dinger1986) [reply update.sh]: sorted on all 3 scripts

File: update.sh (hunk 2 above)
Comment (dinger1986) [reply update.sh]: sorted on all 3 scripts

File: convertfromos.sh [hunk 6]
```diff
@@ -0,0 +1,297 @@
...
+# 5. Download and extract RustDesk Pro Services to the above folder
+# 6. Create systemd services for hbbs and hbbr
+# 7. If you chose Domain, it will install Nginx and certbot, allowing the API to be available on port 443 (https) and get an SSL certificate over port 80, this will autorenew.
+
+# Get Username
+uname=$(whoami)
+admintoken=$(head /dev/urandom | tr -dc A-Za-z0-9 | head -c16)
+
+sudo systemctl stop gohttpserver.service
+sudo systemctl stop rustdesksignal.service
```
Comment (rustdesk) [inline convertfromos.sh:17]: ?
but below is `sudo systemctl enable rustdesk-hbbs.service`

Comment (dinger1986) [reply convertfromos.sh:17]: Yes, if you look at the techahold script for opensource those were the names for the services etc. we need to disable and remove them before creating new ones 😀 

https://raw.githubusercontent.com/techahold/rustdeskinstall/master/install.sh

**Pull request merged**